src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.tsx

```diff
@@ -28,6 +28,17 @@ const Container = styled(Box)(() => ({
   overflowY: 'auto',
 }));
 
+export interface DateRangeFilter {
+  min: string;
+  max: string;
+}
+
+export interface FinancialAccountsFilters {
+  dateRange?: DateRangeFilter;
```

Comment:
Does it work to use the `DateRangeInput` GraphQL type?

```suggestion
  dateRange?: DateRangeInput;
```